.github/workflows/sync-rest.yml

```diff
@@ -151,12 +165,41 @@ jobs:
         repo: ${{ fromJson(needs.fetch-repos.outputs.repos) }}
 
     steps:
+      - name: Check if pull request has already been opened
+        id: pr-already-exists
+        uses: actions/github-script@9ac08808f993958e9de277fe43a64532a609130e
+        env:
+          TARGET_REPO_FULLNAME: ${{ matrix.repo }}
+          TRACKING_ISSUE_URL: ${{ needs.open-tracking-issue.outputs.issue-url }}
+        with:
+          github-token: ${{ secrets.BOT_PERSONAL_ACCESS_TOKEN }}
+          script: |
+            const repo = process.env.TARGET_REPO_FULLNAME.split('/')[1]
+
+            return (
+              await octokit.paginate(octokit.rest.issues.listForRepo, {
+                owner: "exercism",
+                repo: repo,
+                state: 'open',
+              })
+            ).some((pr) =>
+              pr.body.includes(process.env.TRACKING_ISSUE_URL)
+            )
```

Comment:
I'm not familiar with this syntax, could you add a log message here too in case a PR was found?